onnx/tools/replace_constants.py

```diff
@@ -0,0 +1,236 @@
+# SPDX-License-Identifier: Apache-2.0
+# pylint: disable=too-many-statements,too-many-branches
+from typing import List, Optional, Tuple, Union
+
+import numpy as np
+
+from onnx import (
+    AttributeProto,
+    FunctionProto,
+    GraphProto,
+    ModelProto,
+    NodeProto,
+    SparseTensorProto,
+    TensorProto,
+)
+from onnx.helper import (
+    make_attribute,
+    make_function,
+    make_graph,
+    make_model,
+    make_node,
+    make_tensor_value_info,
+    set_model_props,
+    tensor_dtype_to_np_dtype,
+)
+from onnx.numpy_helper import from_array
+
+
+def _replace_constant(
+    node: NodeProto, threshold: int
+) -> Tuple[Optional[NodeProto], NodeProto]:
```

Comment:
Nit: may be simpler and more general to return a `List[NodeProto]`.

Reply:
done